enos/templates/reservation.yaml.sample

```diff
@@ -51,7 +51,9 @@ resources:
 # ############################################### #
 provider:
   type: vagrant
-  box: debian/contrib-jessie64
+  box: bento/debian-9
```

Comment:
We would like to change also in tests/functionnal/packaging,vbox,g5k

Reply:
I do a `find -name 'reservation.yaml` and only found `./tests/functionnal/tests/packaging/reservation.yaml` that use static provider.

Reply:
* https://github.com/BeyondTheClouds/enos/blob/master/tests/functionnal/tests/grid5000/grid5000.yaml
* https://github.com/BeyondTheClouds/enos/blob/master/tests/functionnal/tests/vbox/vbox.yaml

Reply:
Done.

Thanks for the links:
- `grid5000.yaml` does not mention the `env_name`, so it should use the default one.
- I remove `box` from `vbox.yaml`, so it should use the default one.